util/portmuxusr.c

```diff
@@ -497,6 +497,17 @@ static int portmux_get_unix_socket(const char *unix_bind_path)
     socklen_t len;
     int listenfd;
 
+    /*
+    ** gbl_portmux_unix_socket may be changed by clean exit or 'PUT TUNABLE'. We
+    ** store its current value in a local variable and use the variable instead.
+    */
+    char *portmux_unix_socket = gbl_portmux_unix_socket;
```

Comment:
Won't point to freed memory if we have already called clean_exit()?
Do we wont strdup() here?

Reply:
No it won't point to freed memory. It's handled [here](https://github.com/bloomberg/comdb2/pull/2484/files#diff-4f212bc78e2421c5f06542baa07a1660R1799-R1807).

Reply:
clear_portmux_bind_path() can be called from a different thread, right? 
So you may read gbl_portmux_unix_socket here, clear_portmux_bind_path() can go in and free it from the other thread, then you read it in line 550 in this thread. No?

Reply:
Gotcha. You're right.

Reply:
There doesn't seem to be an lightweight approach to handle this. I'm thinking about not freeing portmux unix socket path on clean exit. What do you think?